source/Meadow-Filter-Parse.js

```diff
@@ -148,8 +148,8 @@ const getDataType = (encodedType) =>
 			type = 'UNSIGNED';
 			break;
 		case 'DD':
-			// base 10, 5 decimal places (should cover 99% of use cases)
-			type = 'DECIMAL(10,5)';
+			// precision 40 digits, 5 decimal places (should cover 99% of use cases)
+			type = 'DECIMAL(40,5)';
```

Comment:
not sure what the performance implications are here. 25 is a more realistic default.

Reply:
If we're going crazy with precision, does it make sense to extend this both ways (ex. increase 5 to 15 or something)?

Feels crappy either way. Ideally we'd honor the size of the underlying type, or derive from it.

Reply:
if only we do the underlying type 🙃